packages/components/src/popover/style.scss

```diff
@@ -216,6 +216,7 @@ $arrow-size: 8px;
 
 	.components-popover:not([data-y-axis="middle"])[data-x-axis="left"] & {
 		margin-right: -($grid-unit-30 + $border-width);
+		width: 286px;
```

Comment:
Why exactly this value? Maybe we should use `min-width` to become more flexible?

Reply:
For the exact value choice (https://github.com/WordPress/gutenberg/issues/30532#issuecomment-814180740).

Reply:
This popover component can be used in different areas of WordPress, not just for the date/time component. So we should not hardcode the width.

Reply:
Make sense. Let me try to specify the rule